### PR TITLE
EPMRPP-90984 || Filter projects

### PR DESCRIPTION
**Pull request by maria-hambardzumian**
## PR Checklist

* [x] Have you verified that the PR is pointing to the correct target branch? (`develop` for features/bugfixes, other if mentioned in the task)
* [x] Have you verified that your branch is consistent with the target branch and has no conflicts? (if not, make a rebase under the target branch)
* [x] Have you checked that everything works within the branch according to the task description and tested it locally?
* [x] Have you run the linter (`npm run lint`) prior to submission? Enable the git hook on commit in your IDE to run it and format the code automatically.
* [x] Have you run the tests locally and added/updated them if needed?
* [x] Have you checked that app can be built (`npm run build`)?
* [x] Have you checked that no new circular dependencies appreared with your changes? (the webpack plugin reports circular dependencies within the `dev` npm script)
* [x] Have you made sure that all the necessary pipelines has been successfully completed?
* [ ] If the task requires translations to be updated, have you done this by running the `manage:translations` script?

### CODE REVIEW

File: app/src/common/img/newIcons/filter-filled-inline.svg
Comment (AmsterGet) [inline app/src/common/img/newIcons/filter-filled-inline.svg:1]: Can this state be handled via CSS?
May be also ask UX team to add this icon to DC?

File: app/src/common/urls.js [hunk 1]
```diff
@@ -129,6 +129,8 @@ export const URLS = {
     `${urlCommonBase}organizations${getQueryParams(preferencesObj)}`,
   organizationProjects: (organizationId, preferencesObj = {}) =>
     `${urlCommonBase}organizations/${organizationId}/projects${getQueryParams(preferencesObj)}`,
+  filterOrganizationProjects: (organizationId) =>
```
Comment (AmsterGet) [inline app/src/common/urls.js:132]: May be adjust the name to be more consistent with endpoint name?
e.g. `organizationProjectsSearches`

File: app/src/common/img/newIcons/filter-filled-inline.svg
Comment (maria-hambardzumian) [reply app/src/common/img/newIcons/filter-filled-inline.svg:1]: I tried to use: fill, stroke, but it doesn't seem to work

**Comment by AmsterGet** [comment]
@maria-hambardzumian Can we close this in favor of https://github.com/reportportal/service-ui/pull/4116?

**Comment by maria-hambardzumian** [comment]
Closed in favor of https://github.com/reportportal/service-ui/pull/4116

**Pull request closed without merging**